crates/json-rpc/src/request.rs

```diff
@@ -194,8 +195,8 @@ impl SerializedRequest {
     }
 
     /// Returns the request method.
-    pub const fn method(&self) -> &'static str {
-        self.meta.method
+    pub const fn method(&self) -> &Cow<'static, str> {
```

Comment:
perhaps &str? 
it wouldn't make it const but I think since the caller wants the &str we can just deref here?